lib/ansible/module_utils/urls.py

```diff
@@ -636,8 +636,11 @@ def get_ca_certs(self):
         # and compile them into single temp file for use
         # in the ssl check to speed up the test
         for path in paths_checked:
-            if os.path.exists(path) and os.path.isdir(path):
+            try:
                 dir_contents = os.listdir(path)
+            except OSError:
+                pass
```

Comment:
This should take advantage of https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/common/warnings.py to issue a warning, instead of silently passing now that we have a global facility to add warnings when an instance of `AnsibleModule` isn't available to the code.

Reply:
👍 to this suggestion